src/signed-xml.ts

```diff
@@ -389,6 +394,37 @@ export class SignedXml {
     }
   }
 
+  validateElementAgainstReferences(elem: Element, doc: Document): Reference | false {
```

Comment:
would it make more sense for the return type to be `undefined`, rather than `false`? Alternatively, if most callers would want to throw an Error if invalid, would it make more sense to throw an appropriate Error instead?

Reply:
Done.